layouts/partials/head.html

```diff
@@ -38,5 +38,5 @@
 <link rel="stylesheet" href="{{ "/css/prism.css" | relURL }}"/>
 {{ end }}
 <!-- stylesheet for tabbed pane -->
-<link rel="stylesheet" href="{{ "/css/tabbed-pane.css" | relURL }}"/>
+<link rel="stylesheet" href="{{ "/css/shortcodes.css" | relURL }}"/>
```

Comment:
Just a heads up that this line was deleted (inadvertently it would seem) by a later PR. For details, see #694.